src/averageAges.js

```diff
@@ -14,12 +14,20 @@
  *
  * @return {number}
  */
+function getAverage(people) {
```

Comment:
```suggestion
function getAverageAge(people) {
```
Better specify what exactly function returns